lang/user/id.xml

```diff
@@ -562,24 +564,24 @@ Silakan download dan install firmware PS Vita terakhir.</no_modules>
 		<cpu>
 			<cpu_backend>Bagian Belakang CPU</cpu_backend>
 			<select_cpu_backend>Pilih backend CPU pilihan Kamu.</select_cpu_backend>
-			<cpu_opt>Aktifkan pengoptimalan</cpu_opt>
+			<cpu_opt>Aktifkan Pengoptimalan</cpu_opt>
 			<cpu_opt_description>Centang kotak untuk mengaktifkan optimasi CPU JIT tambahan.</cpu_opt_description>
 		</cpu>
 		<gpu>
-			<backend_renderer>Bagian Belakang Perender</backend_renderer>
+			<backend_renderer>Penyangga Grafis</backend_renderer>
 			<select_backend_renderer>Pilih perender backend pilihan Kamu.</select_backend_renderer>
```

Comment:
<select_backend_renderer>Pilih Penyangga Grafis pilihan Kamu.</select_backend_renderer>

Reply:
Your mean is not for the renderer backend

Reply:
Yes, let me just be the same as the choice

Reply:
i think is not good, you said is graphcis buffer, not renderer backend